packages/contracts/contracts/L1/KromaPortal.sol

```diff
@@ -461,6 +461,35 @@ contract KromaPortal is Initializable, ResourceMetering, Semver {
         emit TransactionDeposited(from, _to, DEPOSIT_VERSION, opaqueData);
     }
 
+    /**
+     * @notice Accepts deposits of data from ValidatorPool contract, and emits a TransactionDeposited event for use in
+     *         deriving deposit transactions.
+     *
+     * @param _to         Target address on L2.
+     * @param _gasLimit   Minimum L2 gas limit (can be greater than or equal to this value).
+     * @param _data       Data to trigger the recipient with.
+     */
+    function depositTransactionByValidatorPool(
+        address _to,
+        uint64 _gasLimit,
+        bytes memory _data
+    ) public {
+        require(
+            msg.sender == address(L2_ORACLE.VALIDATOR_POOL()),
+            "ValidatorRewardPool: function can only be called from the ValidatorPool"
```

Comment:
[Error strings MUST have the format `"{ContractName}: {message}"`](https://github.com/kroma-network/kroma/blob/dev/packages/contracts/README.md#errors)
```suggestion
            "KromaPortal: function can only be called from the ValidatorPool"
```